library/client_events.js

```diff
@@ -32,6 +19,13 @@ module.exports = client => {
       return client.errorMessage(message, 'This bot can only be used on official Pinano servers.')
     }
 
+    let user = await client.userRepository.load(message.author.id)
```

Comment:
@pianiststickman Here's an example of using the new code in just one place, everything else still using the old database lib and leaderboard code.